webrender/src/device/gfx/program.rs

```diff
@@ -652,8 +665,8 @@ impl<B: hal::Backend> Program<B> {
                     .into_iter()
                     .chain(iter::once(desc_set_per_frame))
                     .chain(iter::once(desc_set_per_draw))
-                    .chain(desc_set_locals),
-                &[],
+                    .chain(iter::once(desc_set_locals)),
```

Comment:
this isn't right. The descriptor set carrying projection matrix as well as other per-target parameters should have *lower* index than the per-draw one, since it changes at less frequency

Reply:
Fixed the ordering in https://github.com/szeged/webrender/pull/346/commits/d8716f773e0e8c1945042f359fc958aa37712ae9